/export2/home/andreas/oi-userland/components/desktop/zoitechat/build/manifest-i386-zoitechat.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.file=libayatana-appindicator3.so.1 \
        pkg.debug.depend.reason=usr/bin/zoitechat pkg.debug.depend.type=elf \
        pkg.debug.depend.path=lib/64 \
        pkg.debug.depend.path=usr/gcc/14/lib/amd64 \
        pkg.debug.depend.path=usr/lib/64'.
gmake: *** [/export2/home/andreas/oi-userland/make-rules/ips.mk:529: /export2/home/andreas/oi-userland/components/desktop/zoitechat/

--- a/components/desktop/zoitechat/Makefile
+++ b/components/desktop/zoitechat/Makefile
@@ -1,0 +1,62 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Marcel Telka <marcel@telka.sk>
+# Copyright 2020 Michal Nowak
+#
+
+BUILD_STYLE= meson
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		zoitechat
+COMPONENT_VERSION=	2.18.0
+COMPONENT_SUMMARY=	ZoiteChat IRC client
+COMPONENT_PROJECT_URL=	hhttps://zoitechat.org/ttps://hexchat.github.io/
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_HASH= sha256:e186d2d591320ad0a7f7ffecc794d652ecf1aaf545cff899e5556bdb9b01fbd5
+COMPONENT_ARCHIVE_URL=	https://github.com/ZoiteChat/zoitechat/archive/refs/tags/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=		desktop/irc/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= Applications/Internet
+COMPONENT_LICENSE= GPLv2
+COMPONENT_LICENSE_FILE= COPYING
+
+TEST_TARGET= $(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
+
+PATH= $(PATH.gnu)
+
+# zoitechat-zoitechat-2.18.0/src/common/zoitechat.c:685:17: error: implicit declaration of function 'kill' [-Wimplicit-function-declaration]
+CFLAGS += -Wno-error=implicit-function-declaration
+# zoitechat-zoitechat-2.18.0/src/common/outbound.c:1795:14: error: assignment to 'char *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
+CFLAGS += -Wno-error=int-conversion
+# zoitechat-zoitechat-2.18.0/src/common/url.c:153:55: error: 'strcasecmp' undeclared (first use in this function); did you mean 'g_strcasecmp'?
+CFLAGS += $(XPG7MODE)
+
+CONFIGURE_OPTIONS += -Dwith-python=python-$(PYTHON_VERSION)
+CONFIGURE_OPTIONS += -Dwith-lua=false
+# plugins/perl/meson.build:48: WARNING: Project targets '>= 0.55.0' but uses feature introduced in '1.0.0': "in" string operator.
+# plugins/perl/meson.build:95:2: ERROR: Problem encountered: found perl not suitable for plugin
+CONFIGURE_OPTIONS += -Dwith-perl=false
+CONFIGURE_ENV += AR="/usr/gnu/bin/ar"
+
+# Auto-generated dependencies
+REQUIRED_PACKAGES += $(OPENSSL_PKG)
+REQUIRED_PACKAGES += desktop/library/libayatana-appindicator
+REQUIRED_PACKAGES += library/desktop/cairo
+REQUIRED_PACKAGES += library/desktop/gdk-pixbuf
+REQUIRED_PACKAGES += library/desktop/gtk3
+REQUIRED_PACKAGES += library/desktop/pango
+REQUIRED_PACKAGES += library/desktop/xdg/libcanberra
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += library/libarchive
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/libdbus-glib

--- a/components/desktop/zoitechat/manifests/sample-manifest.p5m
+++ b/components/desktop/zoitechat/manifests/sample-manifest.p5m
@@ -1,0 +1,95 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2026 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/zoitechat
+file path=usr/include/zoitechat-plugin.h
+file path=usr/lib/$(MACH64)/pkgconfig/zoitechat-plugin.pc
+file path=usr/lib/$(MACH64)/zoitechat/plugins/checksum.so
+file path=usr/lib/$(MACH64)/zoitechat/plugins/fishlim.so
+file path=usr/lib/$(MACH64)/zoitechat/plugins/python.so
+file path=usr/lib/$(MACH64)/zoitechat/plugins/sysinfo.so
+file path=usr/lib/$(MACH64)/zoitechat/python/_zoitechat.py
+file path=usr/lib/$(MACH64)/zoitechat/python/hexchat.py
+file path=usr/lib/$(MACH64)/zoitechat/python/xchat.py
+file path=usr/lib/$(MACH64)/zoitechat/python/zoitechat.py
+file path=usr/share/applications/net.zoite.Zoitechat.desktop
+file path=usr/share/dbus-1/services/org.zoitechat.service.service
+file path=usr/share/icons/hicolor/48x48/apps/net.zoite.Zoitechat.png
+file path=usr/share/icons/hicolor/scalable/apps/net.zoite.Zoitechat.svg
+file path=usr/share/locale/af/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/am/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ast/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/az/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/be/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/bg/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ca/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/cs/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/da/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/de/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/el/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/en_GB/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/es/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/et/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/eu/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/fi/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/fr/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/gl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/gu/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/hi/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/hu/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/id/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/it/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ja_JP/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/kn/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ko/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/lt/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/lv/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/mk/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ml/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ms/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/nb/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/nl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/no/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pa/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pt/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pt_BR/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ru/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/rw/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sk/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sq/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sr/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sr@latin/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sv/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/th/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/tr/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/uk/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/vi/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/wa/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/zh_CN/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/zh_TW/LC_MESSAGES/zoitechat.mo
+file path=usr/share/man/man1/zoitechat.1
+file path=usr/share/metainfo/net.zoite.Zoitechat.appdata.xml

--- a/components/desktop/zoitechat/patches/ctcp-version.patch
+++ b/components/desktop/zoitechat/patches/ctcp-version.patch
@@ -1,0 +1,78 @@
+--- hexchat-2.14.3/src/common/util.c	2019-12-21 07:43:47.702403000 +0000
++++ hexchat-2.14.3/src/common/util.c	2020-01-02 12:03:02.367758918 +0000
+@@ -61,6 +61,10 @@
+ #endif
+ #endif
+ 
++#if defined (__SVR4) && defined (__sun)
++#include <kstat.h>
++#endif
++
+ char *
+ file_part (char *file)
+ {
+@@ -360,7 +364,7 @@ strip_hidden_attribute (char *src, char
+ 	return len;
+ }
+ 
+-#if defined (__linux__) || defined (__FreeBSD__) || defined (__APPLE__) || defined (__CYGWIN__)
++#if defined (__linux__) || defined (__FreeBSD__) || defined (__APPLE__) || defined (__CYGWIN__) || (defined (__SVR4) && defined (__sun))
+ 
+ static void
+ get_cpu_info (double *mhz, int *cpus)
+@@ -450,6 +454,37 @@ get_cpu_info (double *mhz, int *cpus)
+ 	*mhz = (freq / 1000000);
+ 
+ #endif
++#if defined (__SVR4) && defined (__sun)
++
++	kstat_t       *ksp;
++	kstat_ctl_t   *kc;
++	kstat_named_t *knp = NULL;
++	int	      numcpu = 0;
++	kc = kstat_open();
++	if (kc == NULL) {
++		fprintf (stderr, "get speed: kstat error %d\n", errno);
++		return;
++	}
++
++	ksp = kstat_lookup(kc, "cpu_info", -1, NULL);
++	for (; ksp != NULL; ksp = ksp->ks_next) {
++                if (strcmp(ksp->ks_module, "cpu_info"))
++                        continue;
++	        numcpu++;
++
++	        kstat_read(kc, ksp, NULL);
++
++		if (knp == NULL) {
++		        knp = kstat_data_lookup(ksp, "clock_MHz");
++		}
++	}
++
++	*cpus = numcpu;
++	if (knp) {
++		*mhz = knp->value.i32;
++	}
++
++#endif
+ 
+ }
+ #endif
+@@ -495,7 +530,7 @@ get_sys_str (int with_cpu)
+ char *
+ get_sys_str (int with_cpu)
+ {
+-#if defined (__linux__) || defined (__FreeBSD__) || defined (__APPLE__) || defined (__CYGWIN__)
++#if defined (__linux__) || defined (__FreeBSD__) || defined (__APPLE__) || defined (__CYGWIN__) || (defined (__SVR4) && defined (__sun))
+ 	double mhz;
+ #endif
+ 	int cpus = 1;
+@@ -507,7 +542,7 @@ get_sys_str (int with_cpu)
+ 
+ 	uname (&un);
+ 
+-#if defined (__linux__) || defined (__FreeBSD__) || defined (__APPLE__) || defined (__CYGWIN__)
++#if defined (__linux__) || defined (__FreeBSD__) || defined (__APPLE__) || defined (__CYGWIN__) || (defined (__SVR4) && defined (__sun))
+ 	get_cpu_info (&mhz, &cpus);
+ 	if (mhz && with_cpu)
+ 	{

--- a/components/desktop/zoitechat/patches/fix-de-gtk.patch
+++ b/components/desktop/zoitechat/patches/fix-de-gtk.patch
@@ -1,0 +1,20 @@
+--- zoitechat-zoitechat-2.18.0/src/fe-gtk/meson.build.~1~
++++ zoitechat-zoitechat-2.18.0/src/fe-gtk/meson.build
+@@ -86,7 +86,7 @@
+   endif
+ endif
+ 
+-zoitechat_gtk_ldflags = []
++zoitechat_gtk_ldflags = ['-lkstat', '-lsocket', '-lnsl', '-lssp', '-lssp_nonshared']
+ 
+ if host_machine.system() == 'windows'
+   zoitechat_gtk_sources += 'notifications/notification-windows.c'
+@@ -134,7 +134,7 @@
+   link_with: zoitechat_theme_lib,
+   c_args: zoitechat_gtk_cflags,
+   link_args: zoitechat_gtk_ldflags,
+-  pie: true,
++  pie: false,
+   install: true,
+   gui_app: true,
+ )

--- a/components/desktop/zoitechat/patches/unknown-system.patch
+++ b/components/desktop/zoitechat/patches/unknown-system.patch
@@ -1,0 +1,11 @@
+--- hexchat-2.14.3/plugins/sysinfo/meson.build	2019-12-21 07:43:47.652401700 +0000
++++ hexchat-2.14.3/plugins/sysinfo/meson.build.new	2020-01-02 12:27:17.764611458 +0000
+@@ -47,7 +47,7 @@ elif system == 'windows'
+     '../../src/common/sysinfo/win32/backend.c'
+   ]
+ else
+-  error('sysinfo: Unknown system?')
++  warning('sysinfo: Unknown system?')
+ endif
+ 
+ shared_module('sysinfo', sysinfo_sources,

--- a/components/desktop/zoitechat/pkg5
+++ b/components/desktop/zoitechat/pkg5
@@ -1,0 +1,19 @@
+{
+    "dependencies": [
+        "desktop/library/libayatana-appindicator",
+        "library/desktop/cairo",
+        "library/desktop/gdk-pixbuf",
+        "library/desktop/gtk3",
+        "library/desktop/pango",
+        "library/desktop/xdg/libcanberra",
+        "library/glib2",
+        "library/libarchive",
+        "library/security/openssl-3",
+        "system/library",
+        "system/library/libdbus-glib"
+    ],
+    "fmris": [
+        "desktop/irc/zoitechat"
+    ],
+    "name": "zoitechat"
+}

--- a/components/desktop/zoitechat/zoitechat.p5m
+++ b/components/desktop/zoitechat/zoitechat.p5m
@@ -1,0 +1,95 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2026 Geoff Weiss
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/zoitechat
+file path=usr/include/zoitechat-plugin.h
+file path=usr/lib/$(MACH64)/pkgconfig/zoitechat-plugin.pc
+file path=usr/lib/$(MACH64)/zoitechat/plugins/checksum.so
+file path=usr/lib/$(MACH64)/zoitechat/plugins/fishlim.so
+file path=usr/lib/$(MACH64)/zoitechat/plugins/python.so
+file path=usr/lib/$(MACH64)/zoitechat/plugins/sysinfo.so
+file path=usr/lib/$(MACH64)/zoitechat/python/_zoitechat.py
+file path=usr/lib/$(MACH64)/zoitechat/python/hexchat.py
+file path=usr/lib/$(MACH64)/zoitechat/python/xchat.py
+file path=usr/lib/$(MACH64)/zoitechat/python/zoitechat.py
+file path=usr/share/applications/net.zoite.Zoitechat.desktop
+file path=usr/share/dbus-1/services/org.zoitechat.service.service
+file path=usr/share/icons/hicolor/48x48/apps/net.zoite.Zoitechat.png
+file path=usr/share/icons/hicolor/scalable/apps/net.zoite.Zoitechat.svg
+file path=usr/share/locale/af/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/am/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ast/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/az/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/be/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/bg/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ca/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/cs/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/da/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/de/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/el/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/en_GB/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/es/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/et/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/eu/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/fi/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/fr/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/gl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/gu/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/hi/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/hu/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/id/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/it/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ja_JP/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/kn/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ko/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/lt/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/lv/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/mk/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ml/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ms/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/nb/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/nl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/no/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pa/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pt/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/pt_BR/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/ru/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/rw/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sk/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sl/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sq/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sr/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sr@latin/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/sv/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/th/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/tr/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/uk/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/vi/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/wa/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/zh_CN/LC_MESSAGES/zoitechat.mo
+file path=usr/share/locale/zh_TW/LC_MESSAGES/zoitechat.mo
+file path=usr/share/man/man1/zoitechat.1
+file path=usr/share/metainfo/net.zoite.Zoitechat.appdata.xml

